static/app/utils/replays/replayDataUtils.tsx

```diff
@@ -51,7 +51,7 @@ export function breadcrumbFactory(
   rawCrumbs: ReplayCrumb[],
   spans: ReplaySpan[]
 ): Crumb[] {
-  const initialUrl = replayRecord.tags.url;
+  const initialUrl = replayRecord.tags.url?.join(', ');
```

Comment:
In your screenshot it looks like we have duplicate URLs -- should the backend be de-duplicating these? (if not we should probably dedupe on UI).

Reply:
Oh! Don't mind it, it was just mocked data that I used for testing and repeated the value